discovery/src/test/java/io/airlift/discovery/client/TestServiceTypes.java

```diff
@@ -48,7 +53,12 @@ public void testValue()
     @Test
     public void testToString()
     {
+        if (System.getProperty("java.version").startsWith("1.8.")) {
+            throw new SkipException("Incompatible Java version: " + System.getProperty("java.version"));
```

Comment:
We should test this on both versions.

Reply:
This test verifies `ServiceTypeImpl#toString()` is consistent with a toString for JVM-created annotation, but this baseline changed.
Not sure what you actually expect here.

Reply:
Ah, I see what you mean. We'd need to make the `toString()` version dependent, which doesn't seem worthwhile.

Reply:
BTW we will need to update this again for Java 14
```
AssertionError: expected [@io.airlift.discovery.client.ServiceType("apple")] but found [@io.airlift.discovery.client.ServiceType(value="apple")]
```

....